signal2html/html.py

```diff
@@ -26,6 +26,8 @@
     is_missed_call,
     is_outgoing_call,
     is_group_call,
+    is_video_call,
+    is_key_update,
     is_group_ctrl,
     is_group_v2_data,
```

Comment:
``is_video_call`` and ``is_group_v2_data`` seem to be unused

Reply:
Indeed, removed